scheduler/pkg/server/server_status.go

```diff
@@ -163,12 +163,11 @@ func (s *SchedulerServer) SubscribeServerStatus(req *pb.ServerSubscriptionReques
 	}
 }
 
-// TODO - Create a ServerStatusMsg type to disambiguate?
-func (s *SchedulerServer) handleServerEvent(event coordinator.ModelEventMsg) {
+func (s *SchedulerServer) handleServerModelEvent(event coordinator.ModelEventMsg) {
 	logger := s.logger.WithField("func", "handleServerEvent")
```

Comment:
I don't have a strong opinion on this, but it might be worth coming up with different name that would describe this event more clearly. For example, `handleModelInstanceEvent`, or something.

Reply:
changed to `handleModelEventForServerStatus`